lib/tests/2wp.js

```diff
@@ -731,6 +739,44 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should reject and refund a pegout with value exactly below minimum', async () => {
+
+      // Arrange
+
+      // Create a pegin for the sender to ensure there is enough funds to pegout and because this is the natural process
+      const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      const peginValueInSatoshis = btcToSatoshis(0.5);
```

Comment:
Maybe this value could be in function of what we are trying to do?

MIN_PEGOUT_VALUE * 2 for example?

Or maybe have a constant defined, BRIDGE_FUNDING_VALUE or something of the sort.

What do you think?

Reply:
Updated.